### DISCUSSION TITLE
Stable shows less accuracy despite having more 300s?

### DISCUSSION THREAD

**Discussion by nuewink** [ppy/osu#27388] [Q&A] [answered]
Hello. I noticed that I often get better accuracy on lazer, despite having less 300s than a stable score. Here's an example of a lazer score
![image](https://github.com/ppy/osu/assets/57014208/0fc34acd-8fa8-4aeb-9713-26ee943321ce)
It has 592 300s and 2 misses. FYI, there's 92/92 slider tick, 267/267 slider end, 7/11 spinner bonus, and 12/12 spinner spin
Now let's see the stable score
![image](https://github.com/ppy/osu/assets/57014208/fc338cbd-6b3a-4220-bebd-8037f23368d4)
It has 598 300s, and only 1 miss. However, the accuracy is 0.03% lower than the lazer score. I'd like to know how the accuracy is calculated on lazer and why it seems to have a higher value than I'd expect

**Comment by bdach** [comment, marked as answer]
Without classic mod active the slider parts (ticks, repeats, tails) are counted separately wrt accuracy rather than as part of the entire slider judgement.

If you wish to return to old accuracy then play classic mod.
